### PR TITLE
Show warning message when debuggee and debugger run at different jvms

### PR DESCRIPTION
**Pull request by testforstephen**
Signed-off-by: Jinbo Wang <jinbwan@microsoft.com>

Fix the issue : https://github.com/Microsoft/vscode-java-debug/issues/30

### CODE REVIEW

File: com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/AttachRequestHandler.java [hunk 1]
```diff
@@ -58,6 +59,18 @@ public void handle(Command command, Arguments arguments, Response response, IDeb
                     attachArguments.timeout);
             context.setDebugSession(debugSession);
             logger.info("Attaching to debuggee VM succeeded.");
+
+            // If the debugger and debuggee run at the different JVM platforms, show a warning message.
+            if (debugSession != null) {
+                String debuggeeVersion = debugSession.getVM().version();
+                String debuggerVersion = System.getProperty("java.version");
+                if (!debuggerVersion.equals(debuggeeVersion)) {
+                    String warnMessage = String.format("[Warn] Remote debuggee JVM: %s, but debugger JVM: %s. The source mapping for jre system libraries "
```
Comment (akaroml) [inline com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/AttachRequestHandler.java]: The debugger and the debuggee are running in different versions of JVMs. You could see wrong source mapping results.
Debugger JVM version: ***
Debuggee JVM  version: ***

Comment (testforstephen) [reply com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/AttachRequestHandler.java]: fixed.

**Comment by vitaliis** [comment]
Is "Debuggee JVM" the one bundled together with vscode? And in case of a Linux snap package, the Java distributed together with vscode? Because I keep seeing this warning and don't know how can I change the "Debuggee JVM version".  This "Debuggee JVM version" is different from JVM I have installed locally.

Can I change the "Debuggee JVM version" somehow (to match the java version I have locally)?

**Comment by DovieW** [comment]
@vitaliis yo you ever figured this out?

**Pull request merged**